openassessment/xblock/openassessmentblock.py

```diff
@@ -1008,6 +1008,8 @@ def render_assessment(self, path, context_dict=None):
         if not context_dict:
             context_dict = {}
 
+        context_dict['text_response_editor'] = self.text_response_editor
```

Comment:
I think this is pretty high level on the change but it would have been the same if I were to go through all of the assessment and pass this as context.